### DISCUSSION TITLE
How to delete a list

### DISCUSSION THREAD

**Discussion by TomTomThomas780** [community/community#70125] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I created a list with no use, I cannot delete it. How can I delete it?

**Comment by radar-blip** [comment]
Click on the list you want to delete, then click on the `Edit` button.
There you have options to `Save List` or `Delete List`.

![Screenshot from 2023-10-15 00-25-52](https://github.com/community/community/assets/39196767/b15a2df0-0ac8-48c0-9633-c7b28b0612b6)

**Comment by 0385582671** [comment]
H

**Comment by alif555** [comment]
I am a beginner, therefore I still don't understand this application
